docs/infrastructure/deployment-targets/cloud-target-discovery/index.md

```diff
@@ -4,18 +4,21 @@ description: Cloud resources can be discovered and registered as deployment targ
 position: 100
 ---
 
+Octopus can discover deployment targets during deployments using tags added to your cloud resources. Target discovery takes place during deployment, and is useful when your deployment process creates your target cloud infrastructure before deploying software to it.
+
 :::hint
-The Early Access Preview of the cloud target discovery feature was added in **Octopus 2022.1** and is enabled via **{{Configuration, Features}}**. The preview does not currently include removal of targets when the cloud resource is removed, this is on our roadmap.
+Cloud Target Discovery was introduced in **Octopus 2022.2** for Azure Web Apps and ECS. EAP 
+support for AKS clusters is provided via a feature toggle in **{{Configuration, Features}}**.
```

Comment:
We don't want a line break here.

Reply:
Sorry this is my IDE wreaking havoc! Good catch 👍